__tests__/app.test.js

```diff
@@ -72,79 +72,224 @@ describe("200: /api", () => {
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          articles.forEach((article) => {
+            expect(article).toHaveProperty("author", expect.any(String));
+            expect(article).toHaveProperty("title", expect.any(String));
+            expect(article).toHaveProperty("article_id", expect.any(Number));
+            expect(article).toHaveProperty("topic", expect.any(String));
+            expect(article).toHaveProperty("created_at", expect.any(String));
+            expect(article).toHaveProperty("votes", expect.any(Number));
+            expect(article).toHaveProperty(
+              "article_img_url",
+              expect.any(String)
+            );
+            expect(article).toHaveProperty("comment_count", expect.any(Number));
+          });
+        });
+    });
+    test("should return an array with specific length", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.length).toBe(5);
+        });
+    });
+    test("should ensure articles are sorted by date in descending order", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeSorted({
+            key: "created_at",
+            descending: true,
+          });
+        });
+    });
   });
-  test("should ensure articles are sorted by date in descending order", () => {
-    return request(app)
-      .get("/api/articles")
-      .expect(200)
-      .then(({ body }) => {
-        const { articles } = body;
-        expect(articles).toBeSorted({
-          key: "created_at",
-          descending: true,
+  describe("200: /api/articles?Queries", () => {
+    test("should order articles by default descending", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeSorted({
+            key: "created_at",
+            descending: true,
+          });
+        });
+    });
+    test("should order articles by descending when specified", () => {
+      return request(app)
+        .get("/api/articles?order=DESC")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeSorted({
+            key: "created_at",
+            descending: true,
+          });
+        });
+    });
+    test("should order articles by ascending", () => {
+      return request(app)
+        .get("/api/articles?order=ASC")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeSorted({
+            key: "created_at",
+            ascending: true,
+          });
         });
+    });
+    describe("suite of test for sort_by", () => {
```

Comment:
Whilst i admire how thorough these tests are, we only really need to do one test to show sort_by works! Having a test to sort by each property is a bit overkill, as we're just displaying the same behaviour repeatedly 🙂